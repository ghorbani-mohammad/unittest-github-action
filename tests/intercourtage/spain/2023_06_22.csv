

Puerto: Tarragona,,,,,,

,,Estado de la llegada: Descargando,,,,

"Producto","Toneladas","Origen","Importador","Consignatario","Barco"
"CEBADA","16.500","BULGARIA","SOUFFLET","BERGÉ BERGÉ","KARPATY"
"TRIGO","52.500","BULGARIA","SOUFFLET","BERGÉ BERGÉ","KARPATY"
"MAIZ","32.800","CANADA","BUNGE IBERICA","ALFASHIP EUROPORTS","NARIE"
"TRIGO","33.000","UCRANIA","CEREALES Y HARINAS GARSAN","BERGÉ BERGÉ","DONNA JUDI"
"GUISANTES","16.300","RUSIA","RIERA ROURA","MARMEDSA NOATUM CODEMAR","REK ROYAL"
"TRIGO","40.000","UCRANIA","RIERA ROURA","ALFASHIP NOATUM CODEMAR","ELENA VE"
"TRIGO","23.700","UCRANIA","VITERRA AGRÍCOLA ESPANA","ALFASHIP ERSHIP","ELENA VE"
"HARINA DE SOJA","11.400","BRASIL","VITERRA AGRÍCOLA ESPAÑA","ERSHIP ERSHIP","KNUT OLDENDORFF"
"DDG","4.700","BRASIL","VITERRA AGRÍCOLA ESPANA","ERSHIP ERSHIP","KNUT OLDENDORFF"


,,,Estado de la llegada: En previsión,,,

"Producto","Toneladas","Origen","Importador","Consignatario/Estibador","Barco"
"TRIGO DURO","6.100","ITALIA","VILAFRANQUINA","ERSHIP ERSHIP","NAVIN HARRIER"
"MAIZ","28.500","RUMANIA","OLAM GRAINS","ERSHIP ERSHIP","STERLING SAGA"
"HARINA DE SOJA","17.000","ARGENTINA","CEFETRA IBERICA","ERSHIP ERSHIP","DONOUSA"
"CASCARILLA DE SOJA","15.000","ARGENTINA","CEFETRA IBERICA","ERSHIP ERSHIP","DONOUSA"


Puerto: Barcelona,,,,,,

,,,Estado de la llegada: Descargando,,,

"Producto","Toneladas","Origen","Importador","Consignatario/Estibador","Barco"
"HABAS DE SOJA","66.000","BRASIL","CARGILL","MASIQUES CARGILL","SAKIZAYA FUTURE"
"TRIGO","23.000","RUMANIA","RIERA ROURA","ALFASHIP ERGRANSA","NOTA A"


"Producto","Toneladas","Origen","Importador","Consignatario/Estibador","Barco"
"MAIZ","21.200","RUMANIA","RIERA ROURA","ALFASHIP ERGRANSA","NOTA A"


,,,Estado de la llegada: En previsión,,,

"Producto","Toneladas","Origen","Importador","Consignatario/Estibador","Barco"
"HABAS DE SOJA","50.000","BRASIL","BUNGE IBERICA","ALFASHIP ERGRANSA","KANG MAY"

