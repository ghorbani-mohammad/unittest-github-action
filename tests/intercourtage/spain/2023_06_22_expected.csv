report_date,contributor_lineup_reference_id,commodity_name,commodity_specification_codes,volume,unit,origin_code,destination_code,expected_cargo_loading_date,cargo_loading_date,cargo_discharged_date,cargo_status,carrier_id,carrier_name,carrier_status,carrier_dead_weight,export_location,import_location,export_terminal,import_terminal,eta,etb,etd,ata,atb,atd,agent,charterer,shipper,operator,comments
